libethcore/Transaction.cpp

```diff
@@ -31,6 +31,12 @@ using namespace std;
 using namespace dev;
 using namespace dev::eth;
 
+namespace
+{
+int const c_chainIdInNonceBits = 64;
+u256 const c_nonceLowMask = 0xffffffffffffffff;
```

Comment:
Global does not have to be created because it has local scope to linker `namespace {}`. Compiler usually produces code like `mov -1 to reg0`. The constant is included in the instruction itself.